org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandlerTest.java

```diff
@@ -210,6 +213,29 @@ public void testRegisterDelayedCapability() throws Exception {
 		verify(client, times(9)).registerCapability(any());
 	}
 
+	@Test
+	public void testConfigureJVMs() throws Exception {
+		InitHandler handler = new InitHandler(projectsManager, preferenceManager, server.getClientConnection(), commandHandler);
+		IEclipsePreferences prefs = InstanceScope.INSTANCE.getNode(IConstants.PLUGIN_ID);
+		boolean isWorkspaceInitialized = isWorkspaceInitialized(prefs);
+		try {
+			prefs.putBoolean(IConstants.WORKSPACE_INITIALIZED, false);
+			InitializeParams params = new InitializeParams();
+			Map<String, Object> initializationOptions = new HashMap<>();
+			initializationOptions.put(InitHandler.SETTINGS_KEY, new HashMap<>());
+			params.setInitializationOptions(initializationOptions);
+			assertFalse(isWorkspaceInitialized(prefs));
```

Comment:
This test actually does not check the JVMs have been configured, as it's supposed to

Reply:
Fixed.